db/migrate/20171112174323_remove_polymorphism_from_ratings.rb

```diff
@@ -0,0 +1,32 @@
+class RemovePolymorphismFromRatings < ActiveRecord::Migration[5.1]
+  class Rating < ActiveRecord::Base; end
+
+  def up
+    add_index :ratings, :application_id
```

Comment:
We're definitely not working with big data here, but I've recently come across the `algorithm: :concurrently` option in [this article](https://robots.thoughtbot.com/how-to-create-postgres-indexes-concurrently-in)

Reply:
Neat! 🤘